### PR TITLE
Unhardcode ChronoShift palettes and sequences

### PR DESCRIPTION
**Pull request by Mailaender**
This exposes them to lint testing and modder configuration. Required for https://github.com/OpenRA/ra2/pull/52 since https://github.com/OpenRA/OpenRA/pull/8176.

### CODE REVIEW

File: OpenRA.Mods.RA/Traits/SupportPowers/ChronoshiftPower.cs [hunk 1]
```diff
@@ -26,6 +27,13 @@ class ChronoshiftPowerInfo : SupportPowerInfo
 		[Desc("Seconds until returning after teleportation.")]
 		public readonly int Duration = 30;
 
+		[PaletteReference] public readonly string TargetOverlayPalette = "terrain";
+
+		public readonly string OverlaySpriteGroup = "overlay";
+		[SequenceReference("OverlaySpriteGroup", true)] public readonly string ValidTileSequence = "target-valid-";
```
Comment (Phrohdoh) [inline OpenRA.Mods.RA/Traits/SupportPowers/ChronoshiftPower.cs]: Please name this `ValidTileSequencePrefix`.

```csharp
OpenRA.Game/GameRules/SoundInfo.cs:24:		public readonly string DefaultPrefix = "";
OpenRA.Mods.Common/Traits/Buildings/RepairableBuilding.cs:35:		public readonly string IndicatorPalettePrefix = "player";
OpenRA.Mods.Common/Traits/Infantry/ScaredyCat.cs:27:		[SequenceReference(null, true)] public readonly string PanicSequencePrefix = "panic-";
OpenRA.Mods.Common/Traits/Infantry/TakeCover.cs:36:		[SequenceReference(null, true)] public readonly string ProneSequencePrefix = "prone-";
OpenRA.Mods.Common/Traits/Player/PlaceBeacon.cs:21:		public readonly string PalettePrefix = "player";
OpenRA.Mods.Common/Traits/Render/WithAttackAnimation.cs:28:		[SequenceReference(null, true)] public readonly string ReloadPrefix = null;
OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs:47:		public readonly string BeaconPalettePrefix = "player";
OpenRA.Mods.RA/Effects/GpsDot.cs:26:		public readonly string IndicatorPalettePrefix = "player";
OpenRA.Mods.TS/Traits/Render/WithPermanentInjury.cs:21:		public readonly string InjuredSequencePrefix = "crippled-";
```

Comment (Mailaender) [inline OpenRA.Mods.RA/Traits/SupportPowers/ChronoshiftPower.cs]: Done.

**Comment by abcdefg30** [comment]
Needs a rebase.

**Comment by Mailaender** [comment]
Rebased.

**Comment by abcdefg30** [comment]
Sorry, needs a rebase again.

Other than that, looks good to me. :+1: / :white_check_mark:

**Comment by Mailaender** [comment]
Rebased again.

**Comment by chrisforbes** [comment]
:+1:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/8cde3b7eee61028c40e420c1d4a9822a96ba8b82...78e5ba0d52df140922b97f75d8bb8471bbf17a48)

**Comment by Phrohdoh** [comment]
[Corrected changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/8cde3b7eee61028c40e420c1d4a9822a96ba8b82...a6c4563513bf21e4db6fa060212cb012f0ff0fc0): `ChronoShiftPower` -> `ChronoshiftPower`

**Comment by abcdefg30** [comment]
Ops, thanks for the correction.

**Pull request merged**